router/pump.go

```diff
@@ -118,15 +130,17 @@ func (p *LogsPump) Run() error {
 }
 
 func (p *LogsPump) pumpLogs(event *docker.APIEvents, backlog bool) {
+	debug("pump.pumpLogs() event.ID:", event.ID)
+	debug("pump.pumpLogs() event.Actor.ID:", event.Actor.ID)
 	id := normalID(event.ID)
```

Comment:
I guess we should use `event.ID`. Since `pump.Update` will also get a null `event.Actor.ID`.

```
2016/04/21 01:19:32 pump.pumpLogs() event.ID: cedf1a9662b1
2016/04/21 01:19:32 pump.pumpLogs() event.Actor.ID:
2016/04/21 01:19:32 pump.update() event.Actor.ID:
2016/04/21 01:19:32 pump.pumpLogs(): cedf1a9662b1 started
2016/04/21 01:19:32 pump.pumpLogs() event.ID: b2b5390b3129
2016/04/21 01:19:32 pump.pumpLogs() event.Actor.ID:
2016/04/21 01:19:32 pump.update() event.Actor.ID:
2016/04/21 01:19:32 pump.pumpLogs(): b2b5390b3129 started
```